pkg/vmimage/data/etc/yum.repos.d/td.repo

```diff
@@ -0,0 +1,5 @@
+[treasuredata]
```

Comment:
We ship `fluentd` and `rubygem-fluent-plugin-systemd` RPMs in the Red Hat OpenShift Container Platform (v. 3.11 for x86_64) - is it feasible to use these instead of treasuredata and the build horror?